public/app/features/dashboard/components/TransformationsEditor/TransformationsEditor.tsx

```diff
@@ -312,29 +377,162 @@ class UnThemedTransformationsEditor extends React.PureComponent<TransformationsE
           </Container>
         )}
         {showPicker ? (
-          <VerticalGroup>
-            <Input
-              aria-label={selectors.components.Transforms.searchInput}
-              value={search ?? ''}
-              autoFocus={!noTransforms}
-              placeholder="Add transformation"
-              onChange={this.onSearchChange}
-              onKeyDown={this.onSearchKeyDown}
-              suffix={suffix}
-            />
-
-            {xforms.map((t) => {
-              return (
-                <TransformationCard
-                  key={t.name}
-                  transform={t}
-                  onClick={() => {
-                    this.onTransformationAdd({ value: t.id });
+          <>
+            {config.featureToggles.transformationsRedesign && (
+              <>
+                {!noTransforms && (
+                  <div
+                    role="button"
```

Comment:
![image](https://github.com/grafana/grafana/assets/20999846/f4ef3356-cffb-41c4-baca-0886d7d213bb)

please use a proper `Button` here with `variant="secondary"`, `fill="text"` and `icon="angle-left"`. few reasons:
- consistency with the rest of the UI
- accessibility
- more readable/avoids `div` soup
- should be able to remove probably all of the custom css

Reply:
It's quite different than how our UX designed it, changing with the existing component but will see with our UX if we want to create a new custom component for that instead

Reply:
For now we will mrge with the existing button component.